ui/app/styles/components/variables.scss

```diff
@@ -184,3 +180,14 @@ table.variable-items {
     opacity: 1;
   }
 }
+
+footer {
+  .button.is-primary.save {
+    margin-left: 8px;
+  }
```

Comment:
Adding margin to individual block-level elements within a row is harder to scale than setting things like spacing at the top level. Try this instead and use CSS grid (or flexbox if you're more comfortable with it):

```
footer {
    display: grid;
    grid-auto-columns: max-content;
    grid-auto-flow: column;
    gap: 1rem;
}
```

and you can remove the 8px margin from the button itself.

This has the added benefit of, if we add more buttons later, we don't have to apply new styles to them. Same for if we rename this class in the future.